JavaScript/Markdown_to_Html/script.js

```diff
@@ -0,0 +1,38 @@
+var output = '';
```

Comment:
Avoid the usage of `var`. Alternatively use `let` and `const` as following the ES6 Syntax